### PR TITLE
Stats: Add layout styles for feedback card

### PR DESCRIPTION
**Pull request by a8ck3n**
Related to: https://github.com/Automattic/red-team/issues/155

## Proposed Changes

Adds basic layout styles to the feedback card. Should work correctly from mobile to Desktop screen sizes. Does not yet apply styles to the CTA text or buttons (including spacing and sizing of emoji).

SCR-20240902-oorj

## Why are these changes being made?

Part of the User Feedback project.

pejTkB-1Ce-p2

## Testing Instructions

* Launch the Calypso Live site.
* Visit the Traffic page.
* Enable the feature flag: `?flags=stats/user-feedback`
* Confirm the feedback card is visible at the bottom of the page.
* Confirm it displays correctly when resizing the view.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/stats/feedback/style.scss [hunk 1]
```diff
@@ -4,5 +4,45 @@
 	border-radius: 5px; // stylelint-disable-line scales/radii
 
 	padding: 20px;
-	margin: 32px;
+
+	@media (max-width: 782px) {
```
Comment (kangzj) [inline client/my-sites/stats/feedback/style.scss]: We should probably use the predefined breakpoints. @dognose24 might have more information.

File: client/my-sites/stats/feedback/style.scss [hunk 2]
```diff
@@ -4,5 +4,45 @@
...
+.stats-feedback-card__cta {
+	@media (max-width: 1340px) {
+		margin-bottom: 12px;
+	}
+}
+
+.stats-feedback-card__actions {
+	display: flex;
+	flex-direction: column;
+	// margin: 20px 0;
```
Comment (kangzj) [inline client/my-sites/stats/feedback/style.scss]: I personally would prefer not to have code commented out in shipped code. I understand it's not finished yet, but still it takes up space and not much meaning.

**Review by kangzj — COMMENTED** [review]
Looks good! Any reason it's still a draft?

**Comment by a8ck3n** [comment]
> Looks good! Any reason it's still a draft?

No, no reason other than I overlooked that as I was wrapping up the changes. Sorry about that!

File: client/my-sites/stats/feedback/style.scss [hunk 3]
```diff
@@ -1,8 +1,60 @@
...
+
 .stats-feedback-card {
 	background: var(--studio-white);
 	border: 1px solid var(--studio-gray-5);
 	border-radius: 5px; // stylelint-disable-line scales/radii
 
 	padding: 20px;
-	margin: 32px;
+
+	@media (max-width: 782px) {
```
Comment (dognose24) [inline client/my-sites/stats/feedback/style.scss]:
```suggestion
	@media (max-width: $break-medium) {
```
Yes, there are a bunch of [breakpoints](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_breakpoints.scss#L11) we can use.

File: client/my-sites/stats/feedback/style.scss [hunk 4]
```diff
@@ -1,8 +1,60 @@
...
+		border-radius: 0;
+		border-top: none;
+		border-left: none;
+		border-right: none;
+	}
+
+	display: flex;
+	flex-direction: column;
+
+	@media (min-width: 1340px) {
```
Comment (dognose24) [inline client/my-sites/stats/feedback/style.scss]: Could you provide more information about why the breakpoint here should be `1340px`?

File: client/my-sites/stats/feedback/style.scss [hunk 5]
```diff
@@ -1,8 +1,60 @@
...
+
+	font-family: $font-sf-pro-text;
+	font-size: $font-body-small;
+	font-weight: 400;
+	line-height: 21px;
+	letter-spacing: -0.24px;
+}
+
+.stats-feedback-card__cta {
+	@media (max-width: 1340px) {
```
Comment (dognose24) [inline client/my-sites/stats/feedback/style.scss]: If we really need to use a customized breakpoint, we can define it as a variable at the top of this file.

File: client/my-sites/stats/feedback/style.scss (hunk 3 above)
Comment (a8ck3n) [reply client/my-sites/stats/feedback/style.scss]: Thanks Dognose. I'll test with the breakpoints in that file. Should work well for this use-case.

File: client/my-sites/stats/feedback/style.scss (hunk 5 above)
Comment (a8ck3n) [reply client/my-sites/stats/feedback/style.scss]: This was just based on my visual testing. The screen needs to be pretty wide before we can safely layout the buttons side-by-side.

**Review by kangzj — APPROVED** [review]
LGTM! Thanks Kevin1

One improvement could be when the layout is switched to column, it might look better if the buttons are of the same length and also centered. We could ask designer about it if you are not sure.

image

**Comment by a8ck3n** [comment]
> One improvement could be when the layout is switched to column, it might look better if the buttons are of the same length and also centered. We could ask designer about it if you are not sure.

I looked at this as well. The one issue I see is that column layout is used for mid-width screens which is probably the most common view (covering tablets & laptops) and in that case, it feels oddly balanced to me. I do think it would make sense for smaller screens though.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16630250

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @a8ck3n for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16630250).

**Pull request merged**